### PR TITLE
Updates numbers on code.org/about to reflect EOY 2015 values.

### PR DESCRIPTION
**Pull request by ashercodeorg**
The new numbers are drawn from https://code.org/about/2015.

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/about/index.md [hunk 1]
```diff
@@ -51,13 +51,13 @@ Hour of Code has won the support of both [Republicans and Democrats](https://www
 | Code.org Goal | Accomplishment |
 |------|----------------|
 | Improve diversity in CS | In our online courses, <%= stats['percent_female'] %>% of  students are girls and 37% are black or Hispanic. In our high school classrooms, [34% are girls, and 60% African American or Hispanic](http://codeorg.tumblr.com/post/98856300118/diversity). |
-| Inspire students | Tens of millions have tried the [Hour of Code](/learn). (<%= format_integer_with_commas(stats['number_served']) %> served. 48% female) |
+| Inspire students | Tens of millions have tried the [Hour of Code](/learn). (<%= format_integer_with_commas(stats['number_served']) %> served. 49% female) |
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/about/index.md:54]: We should also change this on code.org/leaderboard

File: pegasus/sites.v3/code.org/public/about/index.md [hunk 2]
```diff
@@ -51,13 +51,13 @@ Hour of Code has won the support of both [Republicans and Democrats](https://www
...
 | Reach classrooms | <%= format_integer_with_commas(stats['number_teachers']) %> teachers have signed up to teach our intro courses on [Code Studio](http://studio.code.org) and <%= format_integer_with_commas(stats['number_students']) %> students are enrolled. |
 | Prep new CS teachers | We've prepared 20,000 new teachers to teach CS across grades K-12. Learn about our [professional development programs](/educate/professional-development).|
 | Change school district curriculum | We've partnered with [100 of the largest school districts](/educate/partner-districts) to [add CS to the curriculum](/educate/curriculum). These districts teach almost 10% of all U.S. students and 15% of Hispanic and African American students. Learn about [becoming a district partner](/educate/districts).|
-| Set up policies to support CS | Policy changed in [16 states](/action) including CA, NY, FL, IL, OH.|
-| Go global | Our courses are available in 40+ languages, used in all 180+ countries. |
+| Set up policies to support CS | Policy changed in [17 states](/action) including CA, NY, FL, IL, OH.|
+| Go global | Our courses are available in 46+ languages, used in all 180+ countries. |
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/about/index.md]: I would do 45+ since it's exactly 46.

File: pegasus/sites.v3/code.org/public/about/index.md (hunk 1 above)
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/public/about/index.md:54]: Will do in another PR.

**Comment by tanyaparker** [comment]
lgtm

File: pegasus/sites.v3/code.org/public/about/index.md (hunk 2 above)
Comment (ashercodeorg) [inline pegasus/sites.v3/code.org/public/about/index.md]: Done.
